microfrontends/app-builder-menu/package-lock.json

```diff
@@ -1,8 +1,11089 @@
 {
   "name": "app-builder-menu",
   "version": "0.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
please, use the node version `14.19.2` for installing dependencies in this project. Regenerate this file once you have set the correct version